### PR TITLE
Swagger enhancement on CDN

### PR DESCRIPTION
**Pull request by csmengwan**
This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [ ] The title of the PR is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [ ] If applicable, the PR references the bug/issue that it fixes.
- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [ ] My spec meets the review criteria:
  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ ] Validation errors from the [Linter extension for VS Code](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/linter.md) have all been fixed for this spec. (**Note:** for large, previously checked in specs, there will likely be many errors shown. Please contact our team so we can set a timeframe for fixing these errors if your PR is not going to address them).
  - [ ] The spec follows the patterns described in the [Swagger good patterns](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-good-patterns.md) document unless the service API makes this impossible.

---
This change is [Reviewable](https://reviewable.io/reviews/azure/azure-rest-api-specs/656)

### CODE REVIEW

File: arm-cdn/2016-10-02/swagger/cdn.json [hunk 1]
```diff
@@ -421,7 +421,7 @@
         ],
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "OK.",
```
Comment (amarzavery) [inline arm-cdn/2016-10-02/swagger/cdn.json]: Shouldn't this have the same format as others
>  "description": "OK. The request has succeeded",

File: arm-cdn/2016-10-02/swagger/cdn.json [hunk 2]
```diff
@@ -476,13 +476,13 @@
...
```
Comment (amarzavery) [inline arm-cdn/2016-10-02/swagger/cdn.json]: same as above

File: arm-cdn/2016-10-02/swagger/cdn.json [hunk 3]
```diff
@@ -476,13 +476,13 @@
...
             "schema": {
               "$ref": "#/definitions/Endpoint"
             }
           },
           "201": {
-            "description": "Created",
+            "description": "Created.",
```
Comment (amarzavery) [inline arm-cdn/2016-10-02/swagger/cdn.json]: same as above

File: arm-cdn/2016-10-02/swagger/cdn.json [hunk 4]
```diff
@@ -1650,7 +1655,15 @@
         },
         "action": {
           "description": "Action of the geo filter.",
-          "type": "string"
+          "enum": [
+		    "Block",
```
Comment (amarzavery) [inline arm-cdn/2016-10-02/swagger/cdn.json]: Just validating: Is the casing of these enum values correct ?

File: arm-cdn/2016-10-02/swagger/cdn.json [hunk 5]
```diff
@@ -370,7 +370,7 @@
         ],
         "responses": {
           "200": {
-            "description": "OK",
+            "description": "OK. The request has succeeded. The request has succeeded",
```
Comment (amarzavery) [inline arm-cdn/2016-10-02/swagger/cdn.json]: repeated sentence.

File: arm-cdn/2016-10-02/swagger/cdn.json [hunk 6]
```diff
@@ -1650,7 +1655,15 @@
         },
         "action": {
           "description": "Action of the geo filter.",
-          "type": "string"
+          "enum": [
+		    "Block",
+            "Allow"
+		  ],
+		  "type": "string",
```
Comment (amarzavery) [inline arm-cdn/2016-10-02/swagger/cdn.json]: @csmengwan - incorrect indentation. replace tabs with spaces. indentation 2 spaces.

File: arm-cdn/2016-10-02/swagger/cdn.json [hunk 7]
```diff
@@ -1650,7 +1655,15 @@
...
+		  "x-ms-enum": {
+			"name": "action",
```
Comment (hovsepm) [inline arm-cdn/2016-10-02/swagger/cdn.json]: >action [](start = 12, length = 6)

Actions is very generic term please change name to more discoverable. e.g. GeoFilterActions

File: arm-cdn/2016-10-02/swagger/cdn.json (hunk 4 above)
Comment (csmengwan) [inline arm-cdn/2016-10-02/swagger/cdn.json]: Yes

**Comment by hovsepm** [comment]
LGTM

**Pull request merged**